torch/utils/data/dataloader.py

```diff
@@ -134,6 +144,23 @@ def pin_memory_batch(batch):
         return batch
 
 
+def _set_SIGCHLD_handler():
+    global _SIGCHLD_handler_set
+    if _SIGCHLD_handler_set:
+        return
+    previous_handler = signal.getsignal(signal.SIGCHLD)
+
+    def handler(signum, frame):
+        _error_if_any_worker_fails()
+        if callable(previous_handler):
+            previous_handler(signum, frame)
+    try:
+        signal.signal(signal.SIGCHLD, handler)
+    except ValueError as _:
+        return  # Windows doesn't support this
```

Comment:
Can you just make it a no-op on windows? Ignoring all ValueErrors seems to be too much